kaldb/src/main/proto/kaldb_search.proto

```diff
@@ -35,6 +38,17 @@ message HistogramBucket {
   double count = 3;
 }
 
+message Aggregations {
+  repeated Aggregation agg = 1;
+}
+
+message Aggregation {
+  string agg_name = 1;
+  string agg_type = 2;
```

Comment:
Must be one of `sum`, `min`, `max`, `avg`, `value_count`, or `percentile_ranks`.